index.js

```diff
@@ -1,41 +1,11 @@
 module.exports = {
-  extends: 'airbnb-base',
-
-  parser: 'babel-eslint',
```

Comment:
Is this config no more necessary?

Reply:
This config it is `common.js` file.

Reply:
@williamcosta here? https://github.com/getninjas/eslint-config-getninjas/pull/5/files#diff-d7716810d825f4b55d18727c3ccb24e6R5 (just a doubt =D)

Reply:
Yes, In fact I was also in doubt about leaving this config in common. Because in my view the ES5 does not need babel.

Reply:
The parser config option isn't on common.js file. You can just add this there...

Reply:
The parser config option isn't on common.js file. You can just add this there...